rsc/src/main/scala/rsc/settings/Settings.scala

```diff
@@ -24,6 +24,14 @@ object Settings {
       args match {
         case "--" +: rest =>
           loop(settings, false, rest)
+        case argsFile +: rest if argsFile.startsWith("@") =>
+          val source = scala.io.Source.fromFile(argsFile.tail)
+          try {
+            val args0 = source.mkString(" ").split("\\s").toList
```

Comment:
This doesn't seem right.

When I try using code like this to read a text file in the Scala REPL, I get a list of zero- or one-character strings, where each non-whitespace character is in its own string and each whitespace character is replace by an empty string. When I use `mkString` instead of `mkString("")`, I get a list where each non-whitespace token is a separate string and each sequence of N whitespace characters is replaced by a sequence of N-1 empty strings.

```
scala> val source = scala.io.Source.fromFile("README.md")
source: scala.io.BufferedSource = <iterator>

scala> val args0 = source.mkString(" ").split("\\s").toList
args0: List[String] = List(M, a, n, a, g, e, d, "", "", D, e, v, e, l, o, p, m, e, n, t, "", "", E, n, v, i, r, o, n, m, e, n, t, "", "", "", "", P, r, e, r, e, q, u, i, s, i, t, e, :, "", "", m, a, c, O, S, "", "", c, o, m, m, a, n, d, -, l, i, n, e, "", "", d, e, v, e, l, o, p, e, r, "", "", t, o, o, l, s, "", "", "", "", F, o, r, "", "", d, o, c, u, m, e, n, t, a, t, i, o, n, ,, "", "", s, e, e, "", "", t, h, e, "", "", d, o, c, s, "", "", d, i, r, e, c, t, o, r, y, "", "", o, r, "", "", v, i, s, i, t, "", "", g, o, /, m, d, e, ., "", "", "", "", D, i, r, e, c, t, o, r, y, "", "", s, t, r, u, c, t, u, r, e, :, "", "", "", "", "", "", c, i, /, "", "", -, "", "", S, c, r, i, p, t, s, "", "", u, s, e, d, "", "", b, y, "", "", t, h, e, "", "", C, o, n, t, i, n, ...
```